types/duckduckgo-images-api/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for duckduckgo-images-api 1.0
+// Project: https://github.com/KshitijMhatre/duckduckgo-images-api
+// Definitions by: Bart Duisters <https://github.com/bartduisters>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function image_search(params: {
+    query: string,
+    moderate: boolean,
```

Comment:
Documentation states that `moderate`, `iterations` and `retries` are all "optional". You could add the `?` to the end of the parameter to make it optional. e.g. `moderate?: boolean`.